CONTRIBUTING.md

```diff
@@ -132,10 +136,6 @@ A: It is not a rare case that changes are introduced in more
 than one commit. So it is important to let reviewers know
 they should not expect any further commits. **C** is for care.
 
-### Merging the PR
```

Comment:
I'd suggest keeping this section to have end-to-end process described. Even though there is a single line like "PR is merged by one of the mentors".

Reply:
@Oodmincheg Please, reflect on the comment above.

Reply:
Add section back(if I understand well)